### ISSUE TITLE
ValueError: too many values to unpack (expected 2)

### ISSUE THREAD

**Issue by christabone** [jfear/sramongo#10]
Hi Justin -- I received this error from last Friday's console command (left it running over the weekend):

```
2017-05-18 14:18:43,835 - INFO - Parsing SRA XML
Traceback (most recent call last):
  File "/home/chrx/.virtualenvs/flybase-local/bin/sra2mongo", line 11, in <module>
    load_entry_point('sramongo==0.0.1', 'console_scripts', 'sra2mongo')()
  File "/home/chrx/.virtualenvs/flybase-local/lib/python3.5/site-packages/sramongo/sra2mongo.py", line 352, in main
    parse_sra(cache)
  File "/home/chrx/.virtualenvs/flybase-local/lib/python3.5/site-packages/sramongo/sra2mongo.py", line 244, in parse_sra
    for xml, runinfo in cache:
ValueError: too many values to unpack (expected 2)
```

**Comment by jfear** [comment]
Sorry, just saw this. 

I have seen this error before, now I need to remember what was the problem. 

Sorry things are not running smoothly

**Comment by christabone** [comment]
No worries at all, I'm sure we'll get it running smoothly in the near future!

**Comment by jfear** [comment]
Ok, I have a clean instance running python 3.5.2 and the latest master of sramongo installed using pip. I am running sra2mongo now and will keep you updated.

**Comment by jfear** [comment]
Unfortunately, everything ran smoothly. I still need to try on a computer outside of NIH network. From looking at the error I would guess again that you are missing the CSV for one of the sra blocks. Can you look at `.cache/sra2mongo/sra` and make sure all of the CSVs are there?

**Comment by christabone** [comment]
Just checked, looks like all CSVs and XMLx are accounted for (and none appear to be empty).

**Comment by jfear** [comment]
cool, looking at it. shooting to push updates tomorrow.

**Comment by jfear** [comment]
can you zip and send me the `.cache`? Having a hard time replicating.

**Comment by christabone** [comment]
Yep, will do. Working on that now.

**Comment by jfear** [comment]
Ok, it looks like `25500.csv` is missing. However, it is probably best to wait until I have my new changes up and running. Running a real world test now PR #14, but will be nice to have you run it once I merge it in.

**Comment by christabone** [comment]
Oh! Good catch, I must have missed it!

Ready and waiting for further real world tests. I'll give it a go once you merge (as you stated).

**Comment by jfear** [comment]
This should have been fixed in PR #14, re-open if still have problems.
